k8_cortx_cloud/get-logs-from-pvc.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+function usage()
+{
+  echo -e "\n** Recover contents of PVC from Non-Running CORTX Containers **\n"
+  echo -e "Usage: \`sh $0 [PVC]\`\n"
+  echo "Optional Arguments:"
+  echo "    -n|--namespace NAMESPACE: K8s namespace that PVC is in (default=default)"
+  echo "    -f|--force:  Force overwrite of output file"
+  exit 1
+}
+
+pvc=
+namespace=default
+force_overwrite=false
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -n|--namespace )
+      namespace="$2"
+      shift 2
+      ;;
+    -f|--force )
+      force_overwrite=true
+      shift 1
+      ;;
+    * )
+      if [[ $1 = -* ]]; then
+        echo "ERROR: Unsupported Option \"$1\"."
+        usage
+      elif [[ -z "${pvc}" ]]; then
+        pvc=$1
+      fi
+      shift 1
+      ;;
+  esac
+done
+
+if [[ -z "${pvc}" ]]; then
+  echo "ERROR: Specify PVC."
+  exit 1
+fi
+
+datestr=$(date '+%Y%m%d.%H%M%S')
+job_name="cortx-log-${pvc}-${datestr}"
+tarfile="${pvc}.tgz"
+
+if [[ -f "${tarfile}" && "${force_overwrite}" == "false" ]]; then
+  printf "%s already exists. -f to overwrite.\n" "${tarfile}"
+  exit 1
+fi
+
+
+cat << EOF | kubectl apply -f - || true
```

Comment:
If the `kube apply` fails, should it really be ignored? I'd think the rest of the script would not function well if the job isn't created?

Reply:
I agree.  I tried to address it, but ran into a shellcheck obstacle.  I think this will have to remain as it is for now.